crates/holochain_integrity_types/src/op.rs

```diff
@@ -192,6 +194,14 @@ pub struct RegisterAgentActivity {
     pub action: SignedActionHashed,
 }
 
+impl Deref for RegisterAgentActivity {
```

Comment:
This sort of feels like a misuse of deref as deref is ment for containers / smart pointers. This should probably be as ref?

Reply:
Oops yeah, this isn't even needed anymore